scripts/tests/chiptest/python_yaml_tester.py

```diff
@@ -0,0 +1,59 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import click
+
+from chip import ChipDeviceCtrl
+from chip.ChipStack import *
+import chip.FabricAdmin
+import chip.CertificateAuthority
+import chip.native
+import chip.yaml
+
+
+@click.command()
+@click.option(
+    '--setup-code',
+    default=None,
+    help='setup-code')
+@click.option(
+    '--yaml-path',
+    default=None,
+    help='yaml-path')
+def main(setup_code, yaml_path):
+    chip.native.Init()
+    chipStack = ChipStack("/tmp/repl-storage.json")
```

Comment:
Please generate an independent storage. This is not the repl, this is for CI tests. Suggest allowing tempfile module to generate a usable name for you.